con-char.c

```diff
@@ -6,9 +6,9 @@
 #include "emu-char.h"  // generic character backend
 
 
-int con_put_char (byte_t c)
+int con_put_char (word_t c)
 	{
-	return char_send (c);
+	return char_send (c & 0xFF);
```

Comment:
Please use explicit parameter rather than obscure bit masking : keep `byte_t` for the character code, add another `byte_t` for the character attribute.

Reply:
Adding a separate parameter for attributes throughout the project is potentially problematic and needs to be researched a bit more for all platforms as well as within the ELKS ROM BIOS. There is still quite a bit going on I'm developing; I would much prefer to add a ATTR_CODEVAL now to eliminate obscure bit masking and deal with your request in a separate PR after more consideration.

Reply:
Removed mask entirely in this piece of code.